javascript/index.js

```diff
@@ -15,6 +15,22 @@ let splits = document.getElementById('splits');
 
 function printTime() {
```

Comment:
the functionality here works good but you already had some functions and method for this purpose. For example the methods getMinutes and getSeconds of the chronometer give you the current time in minutes and seconds, and the method twoDigitsNumber convert those numbers into an easier to read and use format. Here though, you are essentially redoing that logic instead of reusing those methods.

Reply:
See the solution we will share or the one explained in class to see the better (and simpler) approach.